rest-api/common-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v2/AbstractOperationService.java

```diff
@@ -63,10 +77,17 @@ public Set<OperationDetail> getSupportedOperationDetails(final boolean includeUn
         if (includeUnsupported) {
             operationClasses = new HashSet(ReflectionUtil.getSubTypes(Operation.class));
         } else {
-            operationClasses = getSupportedOperations();
+            operationClasses = getSupportedOperations(false);
         }
-        Set<OperationDetail> operationDetails = new HashSet<>();
-
+        Set<OperationDetail> operationDetails = new TreeSet<>((operationDetail1, operationDetail2) -> {
```

Comment:
This code is the same as what was in a previous version of your other PR. Order of operation details is not part of the ticket related to this PR. Could you revert any changes from this PR which overlap with the other one.

Reply:
Is it safe to use an interactive rebase to fix this, or should I cherry pick ?

Reply:
I think you'll need to edit it manually. The commit history doesn't matter so much as it's all squashed prior to merging.

Reply:
I just push some commits.